<ui version="4.0" >
 <class>DateTimeWidget</class>
 <widget class="QWidget" name="DateTimeWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>250</width>
    <height>250</height>
   </size>
  </property>
  <property name="styleSheet" >
   <string notr="true" >#DateTime{
border: 1px solid black;
}</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="labelTime" >
          <property name="text" >
           <string>Time:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="uiTime" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="uiCalendar" >
        <property name="gridVisible" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
